From d4dced9be715e6f1cdc6b52dd7ee39e0b81e5e6a Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Wed, 12 Feb 2020 00:03:01 +0000
Subject: [PATCH 24/63] recovery: Mark as Lineage Recovery

* So many users think they accidetally restored the stock
  Recovery right now, as we haven't brough our custom UI
  up yet. Make it easier to discern.

Change-Id: Ic803be129e30810c9832a31d0067d981cd9795b3
---
 recovery.cpp                                | 2 +-
 recovery_ui/include/recovery_ui/screen_ui.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/recovery.cpp b/recovery.cpp
index 74016bef..596783cb 100644
--- a/recovery.cpp
+++ b/recovery.cpp
@@ -724,7 +724,7 @@ Device::BuiltinAction start_recovery(Device* device, const std::vector<std::stri
 
   std::vector<std::string> title_lines =
       android::base::Split(android::base::GetProperty("ro.build.fingerprint", ""), ":");
-  title_lines.insert(std::begin(title_lines), "Android Recovery");
+  title_lines.insert(std::begin(title_lines), "Lineage Recovery");
   ui->SetTitle(title_lines);
 
   ui->ResetKeyInterruptStatus();
diff --git a/recovery_ui/include/recovery_ui/screen_ui.h b/recovery_ui/include/recovery_ui/screen_ui.h
index 6a125c08..2a5ab4c7 100644
--- a/recovery_ui/include/recovery_ui/screen_ui.h
+++ b/recovery_ui/include/recovery_ui/screen_ui.h
@@ -127,7 +127,7 @@ class TextMenu : public Menu {
   size_t MenuEnd() const;
 
   // Menu example:
-  // info:                           Android Recovery
+  // info:                           Lineage Recovery
   //                                 ....
   // help messages:                  Swipe up/down to move
   //                                 Swipe left/right to select
-- 
2.36.0

